styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -422,6 +449,15 @@ private static void startScheduler(Scheduler scheduler, ScheduledExecutorService
         TimeUnit.SECONDS);
   }
 
+  private static void startSubmissionRateLimiterUpdate(Storage storage, RateLimiter rateLimiter,
+      ScheduledExecutorService exec) {
+    exec.scheduleAtFixedRate(
+        guard(() -> updateSubmissionRateLimiter(storage, rateLimiter)),
+        SUBMISSION_RATE_LIMITER_UPDATE_TICK_INTERVAL_SECONDS,
```

Comment:
Perhaps not. Was done that way just to be consistent with `startTriggerManager()` and `startScheduler()`. Might not make sense there either.